ftplugin/ada.vim

```diff
@@ -98,6 +110,7 @@ if !exists ("b:match_words")  &&
       \ '\%(\<while\>.*\|\<for\>.*\|'.s:notend.'\)\<loop\>:\<end\s\+loop\>,' .
       \ '\%(\<do\>\|\<begin\>\):\<exception\>:\<end\%(\s*\%($\|;\)\|\s\+\%(\%(if\|case\|loop\|record\)\>\)\@!\a\)\@=,' .
       \ s:notend . '\<record\>:\<end\s\+record\>'
+   let b:undo_ftplugin .= " | unlet! b:match_words"
    let b:match_skip = 's:Comment\|String\|Operator'
```

Comment:
If I correctly understand we concatenate command into `b:undo_ftplugin` that remove all buffer variables again. With the other, already merged, pull request a new variable `b:match_skip` was added. Should that variable be undone as well?

Reply:
Yes, it should.  I'll update this.